webrtc/protocol/simulcast-answer.html

```diff
@@ -57,6 +56,42 @@
   // Check for simulcast attribute with send direction and all RIDs.
   let result = answer_lines.find(
       line => line.startsWith(`a=simulcast:send ${expected_rids.join(';')}`));
-  assert_not_equals(result, undefined, "Could not find simulcast attribute.");
+  assert_not_equals(result, undefined, 'Could not find simulcast attribute.');
 }, 'createAnswer() with multiple send encodings should create simulcast answer');
+
+promise_test(async t => {
+  const pc = new RTCPeerConnection();
+  t.add_cleanup(() => pc.close());
+  const expected_rids = ['foo', 'bar', 'baz'];
+
+  // Try to disable the `bar` encoding by prefixing it with the `~` character.
+  await pc.setRemoteDescription({type: 'offer', sdp: offer_sdp.replace('bar', '~bar')});
+  transceiver.direction = 'sendonly';
+  await pc.setLocalDescription();
+
+  const parameters = pc.getSenders()[0].getParameters();
+  const barEncoding = parameters.encodings.find(encoding => encoding.rid === 'bar');
+  assert_not_equals(barEncoding, undefined);
+  assert_not_equals(barEncoding.active, false);
+}, 'Using the ~rid SDP syntax in a remote description does not control the local encodings active flag');
```

Comment:
Do we need a similar test for answer?
```suggestion
}, 'Using the ~rid SDP syntax in a remote offer does not control the local encodings active flag');
```